eng/scripts/Language-Settings.ps1

```diff
@@ -6,7 +6,35 @@ $packagePattern = "Cargo.toml"
 $GithubUri = "https://github.com/Azure/azure-sdk-for-rust"
 $PackageRepositoryUri = "https://crates.io/crates"
 
-function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
+function SetPackageVersion ($PackageName, $Version, $ServiceDirectory, $ReleaseDate, $ReplaceLatestEntryTitle=$true)
+{
+  if($null -eq $ReleaseDate)
+  {
+    $ReleaseDate = Get-Date -Format "yyyy-MM-dd"
+  }
+  & "$EngDir/scripts/Update-PackageVersion.ps1" -ServiceDirectory $ServiceDirectory -PackageName $PackageName `
+      -NewVersionString $Version -ReleaseDate $ReleaseDate -ReplaceLatestEntryTitle $ReplaceLatestEntryTitle
+}
+
+function GetExistingPackageVersions ($PackageName, $GroupId=$null)
+{
+  try {
+    $PackageName = $PackageName.ToLower()
+    $response = Invoke-RestMethod -Method GET -Uri "https://crates.io/api/v1/crates/${PackageName}/versions"
+    $existingVersions = $response.versions | Sort-Object -Property created_at
```

Comment:
This won't always be the case. We should be sorting based on the semver, which I thought @weshaggard had added support for. We might, for example, publish a patch version after a newer version if customers require.

Reply:
I see `AzureEngSemanticVersion` used below. `num` should be coerced into this type, assuming it supports sorting...which I assume it does.

Reply:
Updated